.gitignore

```diff
@@ -18,6 +18,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
+src/holofuel/__integration_tests__/holo/snapshots/*
```

Comment:
@JettTech This doesn't mean the directory is committed. You have to add a .gitignore file to the directory you want to be committed, so `src/holofuel/__integration_tests__/holo/snapshots/` in this case.